### PR TITLE
contribution form added

### PR DESCRIPTION
**Pull request by chantellerobinson**
Added a contribution form, please see below. Any issues let me know.

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		<div class="form-group">
+			<label class="control-label col-sm-12" for="political-projects">Do you have any connections to other political projects? (optional)</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="political-projects" id="political-projects" placeholder="If you would like to, please let us know of any other political projects you are involved with.">
+			</div>
+        </div>
+        <br>
+        <div class="form-group">
+            <div class="col-sm-offset-2 col-sm-12">
+              <button type="submit" class="btn btn-default">Submit</button>
```
Comment (ghost) [inline form.html]: Maybe this could be a `btn-primary` so it doesn't blend in with the background as much?

File: form.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
```
Comment (ghost) [inline form.html]: With Jekyll, you don't have to copy-paste heading stuff like this into every page -- it's a lot simpler to manage the website if e.g. updating the header only involves updating one file!

So you can get rid of lines 1-99 in this file and simply replace it with:
```
---
title: Get involved
layout: page
---
```

Then Jekyll knows to automatically use the `page` layout (see the file in `_layouts/page.html`)!

File: form.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+			</div>
+        </div>
+        <br>
+        <div class="form-group">
+            <div class="col-sm-offset-2 col-sm-12">
+              <button type="submit" class="btn btn-default">Submit</button>
+            </div>
+          </div>
+    </form>
+
```
Comment (ghost) [inline form.html]: You can also delete all the lines below the `</form>` -- the `page` layout will add those automatically.

**Review by ghost — CHANGES_REQUESTED** [review]
Nice, thank you so much @chantellerobinson ! I've added a few comments, and maybe one other thing (maybe a nitpick) -- instead of having this `form.html`, maybe you could structure the files like `contact/index.html` (i.e. put the file in a folder called `contact`? That way, the URL will look like `wobbly.app/contact` instead of `wobbly.app/form.html` which IMO is a bit neater!

**Comment by ghost** [comment]
Oh, and I forgot -- this pull request is a request to merge your branch (chantellerobinson:redesign) into a new branch called `redesign` in this repo. If you edit the PR to instead request merging into our `master` branch, then we save the step of us merging `redesign` into `master` manually.

**Comment by chantellerobinson** [comment]
Thank you for the detailed response and help! Please be as nitpicking as possible, it's really helpful :)

File: form.html (hunk 1 above)
Comment (chantellerobinson) [reply form.html]: Changed :) Thanks for the tip

File: form.html (hunk 2 above)
Comment (chantellerobinson) [reply form.html]: Amended my code!

**Comment by ghost** [comment]
Thank you @chantellerobinson !  The form looks good :)

However -- it looks like you maybe branched from the old `redesign` branch, which means that merging this PR would also include a lot of old commits that would revert us to an older design. The solution to this is to `git rebase` from our `master` branch, but this is sometimes slightly more involved so can be hard to help with from here -- maybe someone at Code-Operative can help in person?

(Apologies for not catching this earlier, the `redesign` branch is pretty old so I'd forgotten about it)

**Pull request closed without merging**